test/test_json_config.py

```diff
@@ -128,6 +128,121 @@ class TestJsonConfig(unittest.TestCase):
       }
     }'''
 
+    windows_json_file = '''{
+      "config": {
+        "hostname": "mgarewal",
+        "endpoint": "data.logentries.com",
+        "user-key": "3d7946d4-0d97-11e7-9b83-6c0b84a93740",
```

Comment:
Are these keys all valid? Or are they fake (e.g., will not function with the LE, but are adequate for purposes of the tests) ones?